data.json

```diff
@@ -129,10 +129,10 @@
     "name": "Datenanfragen.de e. V.",
     "difficulty": "medium",
     "description": "One merged PR gets at least 3 stickers (limited to 100 sets) and a chance at a limited edition t-shirt.",
```

Comment:
`at least 3 stickers`, is this still valid?
On their website, they call it a `free sticker set`

Reply:
Yup, still valid. I am a maintainer of datenanfragen, so I can definitely say that :)